src/python.rs

```diff
@@ -7,12 +8,19 @@ fn to_html(_py: Python, buffer: String) -> PyResult<String> {
 }
 
 #[pyfunction]
-fn get_offsets(_py: Python, buffer: String) -> PyResult<()> {
+fn get_offsets(_py: Python, buffer: String) -> PyResult<&PyDict> {
```

Comment:
Why is this returning a reference `@PyDict` instead of just passing ownership of the dict?

Reply:
No good reason. This compiles and seems to work, and the alternative complains about the traits of the object.

Reply:
What should be the approach to take here?

Reply:
Not your problem. This goes back to what I was complaining about when I couldn't combine the Lua & Python module generation without writing wrapper functions like this. It has to do with the pyo3 implementation.

Reply:
Yes, this is the kind of issue I was complaining about too when I was forced to pass strings instead of structured data.